overwrites/w3c.json

```diff
@@ -23,10 +23,6 @@
     "service-workers-1": {
         "edDraft": { "replaceWith": "https://w3c.github.io/ServiceWorker/" }
     },
-    "webvr": {
-        "href": { "replaceWith": "https://immersive-web.github.io/webvr/spec/1.1/" },
-        "repository": { "replaceWith": "https://github.com/immersive-web/webxr" }
-    },
     "xpath": {
```

Comment:
The `"xpath"` key and value now also need to be removed from `overwrites/w3c.json`, no?

Reply:
No. Because that one actually is in W3C's database and has to be overwritten, if not, it'll just revert to its previous state within the next hour.